docs/book/src/forc/commands/forc_deploy.md

```diff
@@ -1 +0,0 @@
-# forc deploy
```

Comment:
Should we keep these and write documentation for them? Are they linked to from anywhere?

Reply:
These should be kept they are auto generated by mdbook-preproccesor, which lives [here](https://github.com/FuelLabs/sway/tree/master/scripts/mdbook-forc-documenter)

Reply:
These are unused duplicate pages, there are also ones in the `forc_client` folder.

Reply:
Ahh alright then :)